_includes/configuration/file-picker-callback.md

```diff
@@ -1,14 +1,14 @@
 ## file_picker_callback
 
-This hook can be used to add custom file picker to those dialogs that have it. Internally we support this in *Image*, *Media* and *Link* dialogs. This replaces the [`file_browser_callback`](https://www.tiny.cloud/docs/configure/file-image-upload/#file_browser_callback) (removed in version 5.0) option. The new `file_picker_callback` provides a way to update values of other fields in the dialog.
+This hook can be used to add custom file picker to those dialogs that have it. Internally we support this in *Image*, *Media* and *Link* dialogs. This replaces the [`file_browser_callback`]({{site.baseurl}}/configure/file-image-upload/#file_browser_callback) (removed in version {{site.productname}} 5) option. The new `file_picker_callback` provides a way to update values of other fields in the dialog.
```

Comment:
This should link to v4 docs since file_browser_callback was removed in v5.

Reply:
done